config/api.yml

```diff
@@ -1421,6 +1421,26 @@
       :get:
       - :name: read
         :identifier: event_streams_show
+  :event_definition_sets:
+    :description: Event Definition Sets
+    :identifier: event_definition_set
+    :options:
+    - :collection
+    :verbs: *gp
+    :klass: MiqEventDefinitionSet
+    :subcollections:
+    - :events
+    :collection_actions:
+      :get:
+      - :name: read
+        :identifier: event_definition_set
```

Comment:
I can't find this RBAC identifer...how does this work?

Reply:
This is where the identifier is defined. Then it can be called when it is exposed in the product feature tree, I think. @skateman Can you confirm?
I added the definition here to be consistent with others. Or should we add it only when it is required?

Reply:
@lfu this should be an RBAC feature, I guess we just need this for policy simulation so if @Fryguy agrees, we can just go with `policy_simulation`

Reply:
Updated.